app/src/main/java/com/anytypeio/anytype/ui/editor/EditorFragment.kt

```diff
@@ -949,6 +950,21 @@ open class EditorFragment : NavigationFragment<FragmentEditorBinding>(R.layout.f
                         toast("Couldn't parse url: ${command.url}")
                     }
                 }
+                is Command.OpenAppStore -> {
+                    try {
+                        Intent(Intent.ACTION_VIEW).apply {
+                            data = Uri.parse("${getString(R.string.play_market_url)}${context?.packageName}")
+                        }.let {
+                            startActivity(it)
+                        }
+                    } catch (e: Throwable) {
+                        Intent(Intent.ACTION_VIEW).apply {
```

Comment:
do we need a try/catch block here? might be

Reply:
the code moved to ext function